mitmproxy/flow/export.py

```diff
@@ -4,23 +4,34 @@
 import re
 from textwrap import dedent
 
-from six.moves.urllib.parse import quote, quote_plus
+import six
+from six.moves import urllib
 
 import netlib.http
 
 
+def _native(s):
```

Comment:
`dictstr` feels like a bit hacky to me. Should we replace it with a `json.load` & `json.dumps` pair?